DEV-GUIDE.md

```diff
@@ -0,0 +1,88 @@
+# Simperby Development Guide
+
+## Commit Message Convention
+
+```text
+<title>
+```
+
+or
+
+```text
+<title>
+
+<description>
+```
+
+- `<title>`
+  - summarize your change
+  - use imperative mood.
+  - Omit articles (`a/an` and `the`).
+  - should not exceed 75 characters
+  - must be capitalized
+- `<description>`
+  - detailed explanation of your change
+
+### Examples
+
+```text
+Add account check logic
+
+We have to verify 3 things here:
+1. blah
+2. blah
+3. blah
+```
+
+## Pull Request Convention
+
+- For the title, use imperative mood and make it capitalized.
+- Use **rebase only** (No merge or squash).
+- **NO MERGE COMMITS**.
+- Push your works on your own forked repo, and make a **PR across forks**.
+- Use the same title (head) and content (body) for a single-commit PR.
+- When the PR resolves some issues, put `fixes #123` in the content of the PR, not in the commit messages.
+- Conversation **must be resolved by the commentor**, not the PR author.
+- Do not take the Github notifications as the communication channel; use Discord.
```

Comment:
how about adding spb-dev or spb-eng channel link?

Reply:
This is for the main page. See #82